dotcom-rendering/src/model/buildLightboxImages.ts

```diff
@@ -18,14 +19,32 @@ import { isImage } from './enhance-images';
  *
  */
 const isLightboxable = (
-	element: ImageBlockElement,
+	element: ImageBlockElement | CartoonBlockElement,
 	orientation: Orientation,
 ): boolean => {
 	switch (orientation) {
 		case 'landscape':
-			return element.media.allImages.some(isWideEnough);
+			if (
+				element._type ===
+				'model.dotcomrendering.pageElements.ImageBlockElement'
+			) {
+				return element.media.allImages.some(isWideEnough);
+			} else {
```

Comment:
Assuming this isn't a fallback, but rather `CartoonBlockElement` - is it worth adding the `else if(cartoonblockelement)` to make it explicit / easier to work out without looking at the types?

Reply:
Sounds good 👍